### PR TITLE
Mainnet: readme and changelog updates

### PR DESCRIPTION
**Pull request by mnaamani**
Update reference to chainspec for mainnet in READMEs and bumped the runtime CHANGELOG

### CODE REVIEW

File: runtime/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+### Version 12.1000.0 - Mainnet
+  - Disabled channel payouts proposal
```
Comment (Lezek123) [inline runtime/CHANGELOG.md:2]: Should we add `Renamed Gateway working group to App`?

**Review by Lezek123 — DISMISSED** [review]
LGTM

File: runtime/CHANGELOG.md (hunk 1 above)
Comment (mnaamani) [reply runtime/CHANGELOG.md:2]: well spotted

Comment (mnaamani) [reply runtime/CHANGELOG.md:2]: Fixed, and also updated from master (fix for graphql-server dependency on processor)

**Review by Lezek123 — APPROVED** [review]
LGTM

**Pull request merged**